src/drive/storage/types.ts

```diff
@@ -90,27 +90,46 @@ export interface FetchFolderContentResponse {
 
 export interface FileMeta {
   bucket: string;
-  createdAt: Date;
+  createdAt: string;
+  created_at: string;
   deleted: boolean;
-  deletedAt: Date | null;
-  encryptVersion: string;
+  deletedAt: null;
+  encrypt_version: string;
   fileId: string;
-  folder: string | null;
   folderId: number;
-  folderUuid: string;
+  folder_id: number;
   id: number;
-  modificationTime: string;
   name: string;
-  plainName: string;
-  removed: boolean;
-  removedAt: Date | null;
-  size: string;
-  status: string;
+  plain_name: string | null;
```

Comment:
There are some duplicated values in `camelCase` and `snake_case`.
Wouldn't it be better to try to get rid of those arriving in `snake_case` instead of integrating them into the types?

Reply:
> There are some duplicated values in `camelCase` and `snake_case`.
> Wouldn't it be better to try to get rid of those arriving in `snake_case` instead of integrating them into the types?

I have tried but could not, it is for the web file viewer. I had to make changes that affected other components.